examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,109 @@
+"""
+Point Process Tutorial
+======================
+
+This tutorial shows how to generate GBS point process samples and use them to detect outlier
+points in a data set. Point processes are models for generating random point patterns and GBS
+devices can be programmed to operate as special types of point processes that generate clustered
+point patterns. GBS point processes belong to a class of point processes in which the probability
+of generating a specific pattern of points depends on matrix functions of a kernel matrix that
+describes the similarity between the points. Matrix functions that appear in GBS point processes
+are typically permanents and hafnians. In this tutorial we use the permanental point process in which
+the probability of observing a pattern of points depends on the permanent of their corresponding
+kernel matrix. 
+"""
+
+##############################################################################
+# First we need to import the :mod:`~.gbs.points` and :mod:`~.gbs.plot` modules.
+
+from strawberryfields.gbs import points, plot
+
+##############################################################################
+# We define a space in which the GBS point process patterns are generated. This
+# space is referred to as the state space and is defined by a set of points. The
+# point process selects a subset of these points in each sample. Here we create
+# a discrete and homogeneous two-dimensional space containing 400 points as our
+# state space. 
+
+import numpy as np
+R = np.array([(i, j) for i in range(20) for j in range(20)])
+
+##############################################################################
+# The kernel matrix for the points of this discrete space is constructed using 
+# the :func:`~.kernel` function. The :func:`~.kernel` function uses the *radial
+# basis function* (RBF) kernel:
+#
+# .. math::
+#     K_{i,j} = e^{-\|\bf{r}_i-\bf{r}_j\|^2/2\sigma^2},
+#
+# where :math:`\bf{r}_i` are the coordinates of point :math:`i` and :math:`\sigma`
+# is a kernel parameter that determines the scale of the kernel. Points that are much further
+# than a distance :math:`\sigma` from each other lead to small entries of the kernel matrix,
+# whereas points much closer than :math:`\sigma` generate large entries. The parameter
+# :math:`\sigma` is set to 1.0 in this example. For kernel matrices that are 
+# positive-semidefinite, there exist efficient quantum-inspired classical algorithms
+# for permanental point process sampling :cite:`jahangiri2019point`. In this tutorial
+# we restrict ourselves to such positive-semidefinite kernels and we use the quantum-inspired
```

Comment:
should we say that the RBF is one such kernel?

Reply:
RBF is ps if the distance is calculated in Euclidean space. Not sure if RBF applies to other types of distances or not. That is why did not mention that RBF is ps (as a general term).